internal/pkg/githubactions/uninstall.go

```diff
@@ -0,0 +1,29 @@
+package githubactions
+
+import (
+	"context"
+	"log"
+
+	"github.com/mitchellh/mapstructure"
+)
+
+// Uninstall remove GitHub Actions workflows.
+func Uninstall(options *map[string]interface{}) (bool, error) {
+	ctx := context.Background()
+
+	var opt Options
+	err := mapstructure.Decode(*options, &opt)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	for _, pipeline := range workflows {
+		removeFile(&Param{
```

Comment:
Same as above, some error handling is needed.

Reply:
done